src/overlays/actors/ovl_Bg_Spot01_Idohashira/z_bg_spot01_idohashira.c

```diff
@@ -235,15 +235,16 @@ void func_808AB414(BgSpot01Idohashira* this, PlayState* play) {
 }
 
 void func_808AB444(BgSpot01Idohashira* this, PlayState* play) {
-    CsCmdActorAction* npcAction = BgSpot01Idohashira_GetNpcAction(play, 2);
-    u32 action;
-    u32 currentNpcAction;
-
-    if (npcAction != NULL) {
-        action = npcAction->action;
-        currentNpcAction = this->npcAction;
-        if (action != currentNpcAction) {
-            switch (action) {
+    CsCmdActorCue* cue = BgSpot01Idohashira_GetCue(play, 2);
+    u32 nextCueId;
+    u32 currentCueId;
```

Comment:
```suggestion
    u32 curCueId;
```
Is the full word needed? (applies to all instances)

Reply:
cur and cue look pretty close to each other, is really the only reason i chose to not shorten